### ISSUE TITLE
Fix writing content information

### ISSUE THREAD

**Issue by kelliedesigner** [barnardos/design-system#510]
## Description

The 'Writing content' section of the Design System is missing some of the guidance used internally.

## Links and resources

> - [Read this post and linked research about plain English](https://gds.blog.gov.uk/2014/02/17/guest-post-clarity-is-king-the-evidence-that-reveals-the-desperate-need-to-re-think-the-way-we-write/) on the Government Digital Service blog.

> - [Research](http://works.bepress.com/cgi/viewcontent.cgi?article=1000&context=christopher_trudeau) shows 80% of lawyers prefer plain English. For example, 97% preferred ‘among other things’ over the more traditional Latin phrase ‘inter alia’.

> - [Read why we aim for 9](http://contentdesign.london/reading/specifying-a-reading-age-for-web-content/).

**Comment by kelliedesigner** [comment]
**Content principles**

Aims:

- create digital content that people can easily find, use and understand
- emotionally engage users in the Barnardo’s brand

Be consistent, professional and effective in all our content.

You should:

- avoid duplicating content
- write in plain English
- aim for a reading age of about 9. For technical and professional content, aim for 12.
- choose short, simple words
- use the words your users use (We do the research to find the right words before we start writing.)
- write short, clear sentences – 20 words maximum
- avoid jargon
- if using technical terms, explain them
- write in an active voice
- put the most important information first (especially headings, links, bullets and captions)
- make it easy to scan - Consider the signposts they will see, for example: headings and subheads, bulleted lists, images and captions, links
- check content is easy to use and understand - We use ‘readability’ tests as one measure of how easy it is to understand our content. ‘Readability’ tests check written content to predict what level of ‘reading age’ (level of educational reading ability) someone will need to understand our content.

**Comment by kelliedesigner** [comment]
**Jargon**

Look out for:

- words, acronyms and abbreviations the organisation uses internally
- nominalisation (turning verbs into nouns - use the verb instead
- business speak
- legal terminology and phrasing
- marketing speak
- specialist terms that aren’t technical terms

**Comment by kelliedesigner** [comment]
**Checks**

**Checks for style**

We must check every piece of content we write for Barnardos.

Basics:

- spellcheck
- check links are working
- scan the content – can you tell what it’s about with a quick scan?
- check the readability score
- check it has a review date

Check that the content:
- doesn’t duplicate existing content
- has a clear user need
- is in the right format

Titles should be:
- concise (65 characters maximum)
- clear, specific and positive
- active, start with a verb if possible
- unique to the product
- optimised for search (use keywords)
- in sentence case
- in plain English – no jargon

Body text should:

- have the most important information first
- be concise and easy to scan (with sub-heads every 3-5 paragraphs)
- be written in plain English (no jargon) and easy to understand
- use short sentences (maximum 20 words)
- define acronyms and abbreviations the first time they’re used
- explain any technical terms
- meet voice and tone guidelines

**Comment by kelliedesigner** [comment]
**Checks**

**Checks for factual accuracy**

Subject matter experts check for factual accuracy. They don’t need to waste their time and expertise correcting rewriting for style – our other checks will catch those.

Subject matter experts

do:

- check facts and figures
- check links and citations: is external content accurate, reliable, trustworthy?
- suggest technical language or commonly used expressions if you think that they’re missing
- send agreed changes, not questions or unresolved comments
- provide reasons for changes, so writers can understand
- respond by the deadline

Don’t:

- check for basic mistakes like punctuation errors – we have other checks for that
- correct style – capitals, acronyms, bullets
- revert to text you provided – it’s been changed for a reason

**Comment by kelliedesigner** [comment]
**Maintenance**

We check and iterate content.

All content we publish has a review date. At that date, we check it to see if:

- people are using it, and using it right
- it still meets user need
- it’s correctly positioned for user journey (we may have published something else that will alter this)
- it’s factually accurate
- any external links need to be updated
- the evidence on language is still correct (or are people calling this something else now?)

**Comment by kelliedesigner** [comment]
**Resources**

We can check the reading age of our content using one of the following:

- [the Hemingway app](http://www.hemingwayapp.com/)
- [Microsoft Word](https://support.office.com/en-gb/article/Test-your-document-s-readability-85b4969e-e80a-4777-8dd3-f7fc3c8b3fd2)
- [Readable](https://readable.io/)

**Comment by olliewright** [comment]
Should Margot and Jo be involved in writing the 'Writing content' section
of the Design System - because they have created guidelines for writing the
blog, transformation website, the intranet and external comms?

**Comment by kelliedesigner** [comment]
@olliewright 
For the first iteration of this section, we took Barnardo's Writing Guidelines document and converted it into concise standards.

The acceptance criteria of this ticket to do a second iteration by including the writing guidelines created by the content team for the website, which can be see in all the comments above.

Any further iterations the comms team would like to do can be treated as separate issues.
